crates/turbopack-dev/js/src/runtime.nodejs.js

```diff
@@ -13,11 +13,16 @@ let BACKEND;
       }
 
       if (params.runtimeModuleIds.length > 0) {
-        for (const otherChunkPath of params.otherChunks) {
-          loadChunk(otherChunkPath, {
-            type: SourceTypeRuntime,
-            chunkPath,
-          });
+        for (const otherChunkData of params.otherChunks) {
+          loadChunk(
+            typeof otherChunkData === "string"
+              ? otherChunkData
+              : otherChunkData.path,
```

Comment:
Should be a function `getChunkPath(chunkData)`, we do this very often.

Reply:
It's only used once in this file